src/common/actions/repositories.js

```diff
@@ -53,16 +62,31 @@ export function fetchUserRepositories() {
             throw getError(response, result);
           }
           dispatch(setRepositories(result.payload.repos));
+          dispatch(fetchRepositoriesSuccess());
+          dispatch(setPageLinks(result.pageLinks));
```

Comment:
Page links are supplied by the service (which are in-turn supplied from the GitHub API) to make stateful pagination in the client unnecessary.